raiden/ui/cli.py

```diff
@@ -260,10 +267,35 @@ def options(func):
                 type=str,
                 show_default=True,
             ),
+            option(
+                '--pathfinding-eth-address',
```

Comment:
Damn this is way too many options for the PFS already. I am already confused. As discussed yesterday in the meeting perhaps it's time to also update the docs to explain these new options?

Reply:
Makes sense. Since there will be another pr for issue #3461 I can add the documentation there.